frontend/src/app/spindle/spindle-notation/spindle-notation.component.scss

```diff
@@ -0,0 +1,19 @@
+dt {
+    font-weight: bold;
+}
+
+.mono {
+    font-family: monospace;
+}
+
+var {
```

Comment:
First time I see a use for the `<var>` element! Nice :)

Reply:
I also learned about it just today